core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/projection/ProjectionOpDesc.scala

```diff
@@ -20,6 +22,11 @@ import edu.uci.ics.texera.workflow.common.workflow.{
 
 class ProjectionOpDesc extends MapOpDesc {
 
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Drop Or Keep The Selected Attributes")
+  @JsonPropertyDescription("Choose to drop or keep the selected attributes")
+  var dropOption: DropOption = DropOption.keep
```

Comment:
change this to a boolean, and a checkbox on the frontend. make it default to false (keep).